packages/gator-permissions-snap/.eslintrc.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
       files: ['*.test.ts', '*.test.tsx'],
       rules: {
         '@typescript-eslint/unbound-method': 'off',
+        '@typescript-eslint/no-floating-promises': 'off',
```

Comment:
I see that this covers a bunch of unawaited promises in userEventDispatcher.test.ts - can we either await the promises, or disable this rule just for those cases explicitly?

I think this rule is valuable for catching actual code smells.

Reply:
This is applied only for test files so it the code we still detect it. But I can move to ignore specifically in that test file and still leave it on for others. We can't await them since we are testing if they will be handled in sequence in the case of parallel triggered events.

Reply:
Moved it only to the specific test file